modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/BaseNotificationTypeTest.java

```diff
@@ -138,6 +149,11 @@ public static void setUpClass() throws Exception {
 
 		user1 = TestPropsValues.getUser();
 
+		dtoConverterContext = new DefaultDTOConverterContext(
+			false, Collections.emptyMap(),
+			BaseNotificationTypeTest._dtoConverterRegistry, null,
+			LocaleUtil.getDefault(), null, BaseNotificationTypeTest.user1);
```

Comment:
Hey @guilhermedcamacho, can we replace `BaseNotificationTypeTest.user1` with `user1`?